dat/Commodore - CD32.dat

```diff
@@ -0,0 +1,234 @@
+clrmamepro (
+	name "Commodore - CD32"
+	description "Commodore - CD32"
+	version "2024.11.23"
+	homepage "http://github.com/robloach/libretro-dats"
```

Comment:
``` suggestion
	homepage "http://github.com/libretro/libretro-database"
```

Reply:
Thx, fixed.
If you know who manages the CD32 thumbnails please let me know. There's been a pull request hanging there for 3 weeks. If you guys don't like it or something just let me know what to fix/change. Thx!

Reply:
Thanks for the ping on that. Merged. Addded you to the libretro-thumbnails org too, so feel free to merge things that make sense for CD32.

Reply:
thx